BruteShark/BruteSharkCli/BruteSharkCli.cs

```diff
@@ -122,6 +131,7 @@ private void UpdateAnalyzingStatus()
                 Console.WriteLine($"\r[+] TCP Sessions Analyzed: {_tcpSessionsCount}" + $" UDP Streams Analyzed: {_udpStreamsCount}");
                 Console.WriteLine($"\r[+] Passwords Found: {_passwords.Count}");
                 Console.WriteLine($"\r[+] Hashes Found: {_hashes.Count}");
+                Console.WriteLine($"\r[+] Network Connections Found: {_connections.Count}");
                 Console.SetCursorPosition(0, Console.CursorTop - 4);
```

Comment:
You have added a new line, therefore should be Console.CursorTop - 5
also change to Console.CursorTop + 5 on line 236.
After fixing this the printing will look better:
![image](https://user-images.githubusercontent.com/18364847/100026603-ca580100-2df3-11eb-9315-a34c77c8a7c8.png)